### PR TITLE
Docs: Update manual-writing section

### PR DESCRIPTION
**Pull request by cmills1095**
The "write a section in the manual" part of writing a cookbook is now outdated so I figured I'd try to start an update to accommodate the new formatting. I also updated quickref to be more clear for new cookbook authors.

### CODE REVIEW

File: doc/sphinx/user/extending/write-a-cookbook/manual-section.md [hunk 1]
```diff
@@ -1,63 +1,59 @@
...
+[cookbooks/geophysical-setups.md](https://github.com/geodynamics/aspect/blob/main/doc/sphinx/user/cookbooks/geophysical-setups.md),
+or [benchmarks/index.md](https://github.com/geodynamics/aspect/blob/main/doc/sphinx/user/benchmarks/index.md).
+Your toctree entry should follow the structure
+`cookbooks/YOUR-COOKBOOK-NAME/doc/YOUR-COOKBOOK-NAME.md` or
+`benchmarks/YOUR-COOKBOOK-NAME/doc/YOUR-COOKBOOK-NAME.md` depending on which top-level
+directory your cookbook is going in (see the other toctree entries for examples).
 
-To give you some guidelines on how to write the section in the manual, you can
-follow this general structure:
+Now that it has a place to live in the manual, you'll need to actually create
```
Comment (gassmoeller) [inline doc/sphinx/user/extending/write-a-cookbook/manual-section.md:23]:
```suggestion
Now that it has a place to live in the manual, you will need to actually create
```

**Review by gassmoeller — COMMENTED** [review]
Looks good, but you acquired some conflicts when I merged an earlier PR. Please rebase and address my one comment. Afterwards this is good to go.

**Comment by cmills1095** [comment]
Re-submitted with a short blurb at the end about previewing one's documentation. Ready for re-review

**Review by gassmoeller — APPROVED** [review]

File: doc/sphinx/user/extending/write-a-cookbook/manual-section.md [hunk 2]
```diff
@@ -88,18 +84,18 @@ follow this general structure:
...
-resolution version) of your cookbook.
-
-:::{admonition} TODO
-:class: error
-
-This section needs substantial updating by C. Mills to update the process on how to add a section to the manual.
-A number of relative links also need to be added.
-:::
+{ref}`sec:extending:writing-tests`) that only runs the first time step (or a
+lower resolution version) of your cookbook.
```
Comment (gassmoeller) [inline doc/sphinx/user/extending/write-a-cookbook/manual-section.md:101]:
```suggestion
lower resolution version) of your cookbook.

```

**Pull request merged**